F/Fontconfig/build_tarballs.jl

```diff
@@ -65,18 +64,18 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     HostBuildDependency("gperf_jll"),
-    Dependency("FreeType2_jll"; compat="2.10.4"),
+    Dependency("FreeType2_jll"; compat="2.13.3"),
     Dependency("Bzip2_jll"; compat="1.0.8"),
     Dependency("Zlib_jll"),
     Dependency("Libuuid_jll"),
-    Dependency("Expat_jll"; compat="2.2.10"),
+    Dependency("Expat_jll"; compat="2.6.4"),
 ]
 
+init_block = """
+get!(ENV, "FONTCONFIG_FILE", fonts_conf)
+get!(ENV, "FONTCONFIG_PATH", dirname(ENV["FONTCONFIG_FILE"]))
+"""
+
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6",
-               preferred_gcc_version=v"6",
-               init_block = """
-get!(ENV, "FONTCONFIG_FILE", fonts_conf)
-    get!(ENV, "FONTCONFIG_PATH", dirname(ENV["FONTCONFIG_FILE"]))
```

Comment:
I know this looks funky, but it makes code in the JLL indented correctly: https://github.com/JuliaBinaryWrappers/Fontconfig_jll.jl/blob/56f41ffaf42a49e5e5fd9e1a422cab7633fba957/src/wrappers/aarch64-apple-darwin.jl#L79-L81.  What you did will break indentation there.

Reply:
Oh my.